app/assets/stylesheets/prism.scss

```diff
@@ -0,0 +1,159 @@
+
```

Comment:
It looks like GOV.UK have added this to a CDN

https://github.com/alphagov/govuk-frontend/search?p=1&q=prism&type=&utf8=%E2%9C%93